heltour/tournament/models.py

```diff
@@ -1553,7 +1564,7 @@ def other_seasons(self):
         return SeasonPlayer.objects.filter(player__lichess_username__iexact=self.lichess_username).exclude(season=self.season)
 
     def player(self):
-        return Player.objects.filter(lichess_username__iexact=self.lichess_username).first()
+        return self.user.player
```

Comment:
woops, this is wrong.  Should use `Player.get_or_create(self.user)` or something